### PR TITLE
[LOCATION-9] Run handleCountryChange function at first form render

### PR DESCRIPTION
**Pull request by Arturo777**
[Related Ticket](https://vtex-dev.atlassian.net/browse/LOCATION-9)
[Workspace to test](https://arturovtex--emdesign.myvtex.com/gildan-fleece-crewneck-sweatshirt/p?__bindingAddress=www.emdesign.bg/)

Description:
Client is using the shopper-location app and is experiencing a problem on the Bulgaria store. See steps to replicate the problem:

go to a [PDP page](https://arturovtex--emdesign.myvtex.com/holna-garnitura-belle-310x250-sm-kafyavo-i-bejovo/p?__bindingAddress=www.emdesign.ro/), Bulgaria store:
Click on add location on the right pane.
The only available field was the country selector
The only way to display the other fields was to select 'Romain', then 'Bulgarian' again.
All the fields needed to be initially displayed.
Now i put the function in charge to render the fields at the first form render.

### CODE REVIEW

File: react/components/LocationForm.tsx [hunk 1]
```diff
@@ -180,21 +180,18 @@ const LocationForm: FunctionComponent<WrappedComponentProps &
...
-    location.country.value = culture.country
-  }
-
   useEffect(() => {
     isMountedRef.current = true
     currentAddress.receiverName = currentAddress.receiverName || { value: ' ' }
-    const addressWithValidation = addValidation(currentAddress)
+    const country = (location?.country?.value ?? culture.country) || ''
 
     if (isMountedRef.current) {
```
Comment (polishq) [inline react/components/LocationForm.tsx:188]:
```suggestion
    if (isMountedRef.current && country) {
```

File: react/index.tsx [hunk 2]
```diff
@@ -116,7 +116,7 @@ const AddressChallenge: StorefrontFunctionComponent<WrappedComponentProps &
         })
     },
     // eslint-disable-next-line react-hooks/exhaustive-deps
```
Comment (polishq) [inline react/index.tsx:118]: Remove eslint-disable if possible

File: react/index.tsx [hunk 3]
```diff
@@ -175,7 +175,7 @@ const AddressChallenge: StorefrontFunctionComponent<WrappedComponentProps &
           })
       })
     // eslint-disable-next-line react-hooks/exhaustive-deps
```
Comment (polishq) [inline react/index.tsx:177]: Remove eslint-disable if possible

**Review by polishq — CHANGES_REQUESTED** [review]

File: react/index.tsx (hunk 3 above)
Comment (Arturo777) [reply react/index.tsx]: yeah, i didn't made this disable rule comment, but i removed already, i had to add `updateRegionID` to the array dependencies and wrap the function into its own callback function.

File: react/index.tsx (hunk 2 above)
Comment (Arturo777) [reply react/index.tsx]: yeah, i didn't made this disable rule comment, but i removed already, i had to add updateRegionID to the array dependencies and wrap the function into its own callback function.

**Review by polishq — APPROVED** [review]

**Pull request merged**